c7n/manager.py

```diff
@@ -16,12 +16,21 @@
 import logging
 
 from c7n import cache
+from c7n.actions import AutoTagUser
 from c7n.executor import ThreadPoolExecutor
 from c7n.registry import PluginRegistry
 from c7n.utils import dumps
 
 
+def add_auto_tag_user(registry, _):
```

Comment:
i'd suggest moving this to near the auto tags definition,  else its a bit of action at a distance.